internal/spdxlicense/generate/generate_license_list_test.go

```diff
@@ -22,18 +22,18 @@ func TestReplaceDeprecatedLicenses(t *testing.T) {
 	}
 
 	expected := map[string]string{
-		"abc-1":              "ABC-1.0-Only",
-		"abc-1-only":         "ABC-1.0-Only",
-		"abc-1.0":            "ABC-1.0-Only",
-		"abc-1.0.0":          "ABC-1.0-Only",
-		"abc-1.0-only":       "ABC-1.0-Only",
-		"abc-1.0.0-only":     "ABC-1.0-Only",
-		"abc-1+":             "ABC-1.0-Or-later",
-		"abc-1.0+":           "ABC-1.0-Or-later",
-		"abc-1.0.0+":         "ABC-1.0-Or-later",
-		"abc-1-or-later":     "ABC-1.0-Or-later",
-		"abc-1.0-or-later":   "ABC-1.0-Or-later",
-		"abc-1.0.0-or-later": "ABC-1.0-Or-later",
+		"abc-1":              "ABC-1.0-only",
```

Comment:
Interesting that the casing has changed here, but just wanted to confirm for others who come across this that these are correct as of 3.18:

https://spdx.org/licenses/ <-- "or" and "only" are lower cased here

Reply:
Yes, I believe  these should be lower cased. I don't see any license with upper case variants.